spec/spec_helper.rb

```diff
@@ -1,9 +1,7 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 
-if ENV["TEST_SUITE"] == "vmdb"
-  require 'coveralls'
-  Coveralls.wear!('rails') { add_filter("/spec/") }
-end
+require 'coveralls'
```

Comment:
Good question, @Fryguy @matthewd mentioned coveralls uses simplecov which is supposed to have nearly no overhead.  I don't know if running it locally makes any impact.